pkgs/servers/sql/postgresql/default.nix

```diff
@@ -100,9 +118,11 @@ let
         moveToOutput "lib/libpgcommon*.a" "$out"
         moveToOutput "lib/libpgport*.a" "$out"
         moveToOutput "lib/libecpg*" "$out"
+        moveToOutput "bin/pg_config" "$dev"
+        moveToOutput "lib/pgxs/src/Makefile.global" "$dev"
```

Comment:
Looks like more paths (e.g. `/lib/pgxs/src/makefiles/pgxs.mk`) need to be moved and `PGXS` entry in `pg_config` updated.